src/main/java/com/faforever/client/game/CustomGamesController.java

```diff
@@ -239,7 +249,9 @@ void setSelectedGame(Game game) {
       return;
     }
 
-    gameDetailPane.setVisible(true);
+    if (preferencesService.getPreferences().isShowGameDetailsSidePane()) {
+      gameDetailPane.setVisible(true);
```

Comment:
just bind the two properties

Reply:
same as above, if I bind this, the visibility depends on the preference to show the pane or not. But the pane can also be not shown if the preference says it should be shown if there is no game selected.